README.md

```diff
@@ -26,9 +26,10 @@ Please see the link below for more detail on TypeScript 3.8.x:
 * [TypeScript 3.8.x](https://www.npmjs.com/package/typescript)
```

Comment:
Should we specify min Node.JS version here?

Reply:
We'll have to find out the min Node.js version.